### ISSUE TITLE
No such host issues

### ISSUE THREAD

**Issue by Elyytscha** [BeryJu/gravity#932]
i have issues with using this dns but im not sure whats the problem

what i see is that my certmanager gets often no such host for
```
https://acme-v02.api.letsencrypt.org/
```
first i thought lets encrypt is blocking my ip for to many requests or such thing

but what resolved my issue was deleting all records in the root zone

the root zone has the following config
![image](https://github.com/BeryJu/gravity/assets/10408311/fd5f756e-351c-45bb-99f5-230f2f721741)
![image](https://github.com/BeryJu/gravity/assets/10408311/e42e7fa4-a4e4-42c4-bf57-c0b36c372b89)

what i do to (temporarily) resolve:
![image](https://github.com/BeryJu/gravity/assets/10408311/4a2d768f-2940-4dbd-a6c4-ee0a5fa94ad1)

**Comment by Elyytscha** [comment]
i think as a workaround i removed
```
- type: memory
- type: etcd
```
from the root zone, but i'm not sure if thats a good thing, now im living without dns cache i think?

**Comment by BeryJu** [comment]
Most likely the same issue as https://github.com/BeryJu/gravity/issues/838, also fixed with 0.9
